src/ResourceManager/Cdn/Commands.Cdn/Endpoint/RemoveAzureCdnEndnpoint.cs

```diff
@@ -0,0 +1,89 @@
+using Microsoft.Azure.Management.Cdn;
+using Microsoft.Azure.Management.Cdn.Models;
+
+namespace Microsoft.Azure.Commands.Cdn.Endpoint
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureCdnEndpoint", ConfirmImpact = ConfirmImpact.High), OutputType(typeof(bool))]
+    public class RemoveAzureCdnEndpoint : AzureCdnCmdletBase
+    {
+        [Parameter(Mandatory = true, ParameterSetName = FieldsParameterSet, HelpMessage = "Azure Cdn endpoint name.")]
+        [ValidateNotNullOrEmpty]
+        public string EndpointName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = FieldsParameterSet, HelpMessage = "Azure Cdn profile name.")]
+        [ValidateNotNullOrEmpty]
+        public string ProfileName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = FieldsParameterSet, HelpMessage = "The resource group of the Azure Cdn Profile")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = ObjectParameterSet, ValueFromPipeline = true, HelpMessage = "The endpoint.")]
+        [ValidateNotNull]
+        public PSEndpoint CdnEndpoint { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ParameterSetName == ObjectParameterSet)
+            {
+                ResourceGroupName = CdnEndpoint.ResourceGroupName;
+                ProfileName = CdnEndpoint.ProfileName;
+                EndpointName = CdnEndpoint.Name;
+            }
+
+            try
+            {
+                CdnManagementClient.Endpoints.Get(EndpointName, ProfileName, ResourceGroupName);
+            }
+            catch (AggregateException exception)
+            {
+                var errorResponseException = exception.InnerException as ErrorResponseException;
+                if (errorResponseException == null)
+                {
+                    throw;
+                }
+
+                if (errorResponseException.Response.StatusCode.Equals(HttpStatusCode.NotFound))
+                {
+                    throw new PSArgumentException(string.Format(
+                        Resources.Error_DeleteNonExistingEndpoint,
+                        EndpointName,
+                        ProfileName,
+                        ResourceGroupName));
+                }
+            }
+
+            ConfirmAction(
+                        Force.IsPresent,
+                        string.Format(Resources.Confirm_RemoveEndpoint, EndpointName, ProfileName, ResourceGroupName),
+                        () => { CdnManagementClient.Endpoints.DeleteIfExists(EndpointName, ProfileName, ResourceGroupName); });
+
+            WriteVerbose(Resources.Success);
+            WriteVerbose(string.Format(Resources.Success_RemoveEndpoint, EndpointName, ProfileName, ResourceGroupName));
+            WriteObject(true);
```

Comment:
In most cases, you will want to write nothing on success.  Users should be able to pass -PassThrough to get the true for success printed.